src/components/Accordion/AccordionContext.tsx

```diff
@@ -0,0 +1,114 @@
+import * as React from 'react';
+
+import _isEqual from 'lodash/isEqual';
+
+import type {AccordionProps, AccordionValue} from './types';
+import {updateAccordionItemsApi} from './utils';
+
+interface AccordionProviderProps<Multiple extends boolean>
+    extends Required<
+        Omit<
+            AccordionProps<Multiple>,
+            'qa' | 'className' | 'value' | 'defaultValue' | 'multiple' | 'ariaLabel'
+        >
+    > {
+    multiple: Multiple;
+    value: AccordionValue<Multiple>;
+    defaultValue: AccordionValue<Multiple>;
+    onUpdate: (value: AccordionValue<Multiple>) => void;
+    controlled: boolean;
+}
+
+const AccordionAttributesContext = React.createContext<
```

Comment:
Why is it called `Attributes`? It seems you can just leave the `AccordionContext`